scriptoocloud/src/main/webapp/WEB-INF/views/datasets/details.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en"
+      layout:decorate="~{layouts/main}"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      xmlns:th="https://www.thymeleaf.org">
+<head>
+    <title>Dataset Details | ScripToo Cloud</title>
+</head>
+<body>
+<div layout:fragment="content">
+    <div id="wrapper">
+        <section class="container main-page-content ">
+            <div class="row">
+                <h2><strong>Dataset</strong> Details</h2>
+                <div class="alert alert-success"
+                     th:if="${not #strings.isEmpty(successMessage)}">
+                    <p th:text="${successMessage}">
+                </div>
+                <div class="alert alert-danger"
+                     th:if="${not #strings.isEmpty(errorMessage)}">
+                    <p th:text="${errorMessage}">
+                </div>
```

Comment:
what are these used for? for general error messages? I would put them in your layout so they are available on every page.

Reply:
These error and success messages are specific to `Dataset` operations on the `details.html` page.